tests/cql_assertions.cc

```diff
@@ -180,7 +180,7 @@ rows_assertions rows_assertions::with_serialized_columns_count(size_t columns_co
 }
 
 shared_ptr<cql_transport::messages::result_message> cquery_nofail(
-        cql_test_env& env, const char* query, std::unique_ptr<cql3::query_options>&& qo, const std::experimental::source_location& loc) {
+        cql_test_env& env, const seastar::sstring& query, std::unique_ptr<cql3::query_options>&& qo, const std::experimental::source_location& loc) {
```

Comment:
This change fairly harmless to me (a little bit more work for the compiler...), but @dekimir this is code you wrote, can you please check if you're fine with it too?

Reply:
For changing the signature of a test method (verified to compile)? Sure. @dekimir, please review.

Reply:
LGTM (nit: `seastar::` is superfluous here)